duo_client/admin.py

```diff
@@ -3677,7 +3677,7 @@ def get_passport_config(self):
         response = self.json_api_call("GET", path, {})
         return response
 
-    def update_passport_config(self, enabled_status, enabled_groups=[], disabled_groups=[]):
+    def update_passport_config(self, enabled_status, enabled_groups=[], disabled_groups=[], custom_supported_browsers={"macos": [], "windows": [],}):
```

Comment:
Using empty lists in function parameters is against Python convention (https://www.codiga.io/blog/python-avoid-empty-array-parameter/)

The already existing parameters will need to be fixed as well.

Reply:
This is my mistake, I did not catch the issue when this method was introduced with the empty list parameter defaults.